src/constants.ts

```diff
@@ -5,6 +5,8 @@ export const PANEL_ID = `${ADDON_ID}/panel`;
 
 export const {
     STORYBOOK_ZEPLIN_TOKEN: ZEPLIN_TOKEN,
+    STORYBOOK_ZEPLIN_PROJECT_ID: ZEPLIN_PROJECT_ID,
```

Comment:
I tried to use `options` field of `addons` array in `main.js`; however, options are used for presets and couldn't get at runtime. I will try again to use `options`.

I will also add documentation after that.